src/Components/People.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+ class People extends Component {
+  constructor() {
+    super();
+    this.state = {
+      searchPeople: "",
+      currentPeople: {},
+      isError: false,
+      savedName: "",
+    };
+  }
+
+  handleChange = (e) => {
+    this.setState({
+      searchPeople: e.target.value,
+    });
+  };
+
+  handleSubmit = async (e) => {
+    e.preventDefault();
+    const { searchPeople } = this.state;
+    try {
+      const { data } = await axios.get(
+        "https://ghibliapi.herokuapp.com/people"
+      );
+    
+      const person = data.find((element) => element.name === searchPeople);
+    
+      if (person === undefined) {
+        this.setState({
+          searchPeople: "",
+          currentPeople: {},
+          isError: true,
+          savedName: "",
+        });
+        
+      }
+      this.setState({
+        searchPeople: "",
+        currentPeople: person,
+        isError: false,
+        savedName: person.name,
+      });
```

Comment:
It seems like this code will run even if `person === undefined` - is that what you want? Consider putting this in an else block